srcpkgs/libblockdev/template

```diff
@@ -11,9 +11,9 @@ short_desc="Library for manipulating block devices"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/storaged-project/libblockdev"
-changelog="https://raw.githubusercontent.com/storaged-project/libblockdev/master/NEWS.rst"
+changelog="https://raw.githubusercontent.com/storaged-project/libblockdev/${version%%.*}.x-branch/NEWS.rst"
```

Comment:
This breaks copy and pasting the changelog url to the browser so as to help review the changes. Please make this just `2.x-branch` and comment above it "This needs to be adjusted as the major version changes."

I'm hesitant to want 2 versions of changelog=, one commented and one not, since the field was introduced primarily so reviewers could quickly reference a list of changes.